src/popup/locales/cn.json

```diff
@@ -208,10 +208,9 @@
       "noTransactionsFound": "未发现转帐！",
       "balance": "余额",
       "copied": "已复制!",
-      "seedNotification": "你需要 {0} 你的助记词",
+      "seedNotification": "Welcome to Superhero wallet! Please backup your seed phrase!",
```

Comment:
I suggest to remove the entries, that are using english within non-english translation files. This makes no sense to bloat files.

Reply:
https://github.com/aeternity/superhero-wallet/issues/452